config/routes.rb

```diff
@@ -4,6 +4,8 @@
 
   root to: 'spotlight/exhibits#index'
 
+  get '/exhibits_feed', to: 'spotlight/exhibits#index', defaults: { format: :rss }
```

Comment:
I think wordpress standard is to add /feed to the end of a url to get the rss feed. What do you think about doing the same? e.g. `get '/feed', ...` instead of exhibit_feed.